classes/submission/PKPSubmissionFileDAO.inc.php

```diff
@@ -53,12 +53,11 @@ function PKPSubmissionFileDAO() {
 	//
 	/**
 	 * Retrieve a specific revision of a file.
-	 * @param $fileId int
-	 * @param $revision int
-	 * @param $fileStage int (optional) further restricts
-	 *  the selection to a given file stage.
-	 * @param $submissionId int (optional) for validation
-	 *  purposes only
+	 * @param $fileId int File ID.
+	 * @param $revision int File revision number.
+	 * @param $fileStage int (optional) further restricts the selection to
```

Comment:
I tend to copy/paste from similar examples. I suppose we have two ways of saying optional. Doxygen doesn't care; this is part of the description field.